docs/flow/modin/experimental/batch.rst

```diff
@@ -0,0 +1,10 @@
+Batch Pipeline API description
+""""""""""""""""""""""""""""""
+
+This API exposes the ability to pipeline row-parallel batch queries on a Modin DataFrame. Currently,
+this feature is only supported for the ``ray`` Engine.
```

Comment:
```suggestion
this feature is only supported for the ``ray`` engine.
```

Reply:
We can also put a link to https://modin.readthedocs.io/en/stable/development/using_pandas_on_ray.html.